BridgeSDK/BridgeAPI/SBBBridgeNetworkManager.m

```diff
@@ -102,10 +122,47 @@ - (void)handleHTTPError:(NSError *)error task:(NSURLSessionDataTask *)task respo
             }
         }];
     } else {
+        [self checkForAndHandleUnsupportedAppVersionHTTPError:error];
         [super handleHTTPError:error task:task response:responseObject retryObject:retryObject];
     }
 }
 
+- (void)checkForAndHandleUnsupportedAppVersionHTTPError:(NSError *)error
+{
+    // Set flag that blocks attempting further retries once this error has been received.
+    // All future attempts to access services will fail.
+    if (error.code == SBBErrorCodeUnsupportedAppVersion && !self.isUnsupportedAppVersion)
+    {
+        // Set flag that this exception has already been thrown by the server
+        _unsupportedAppVersion = YES;
+        
+        // Look to see if the app delegate handles this error or if this SDK should do so.
+        // Note: check conforms to protocol to ensure that the app delegate is intentionally
+        // implementing 
+        id appDelegate = [[UIApplication sharedApplication] delegate];
+        if (![appDelegate conformsToProtocol:@protocol(SSBBridgeAppDelegate)] ||
+            ![appDelegate respondsToSelector:@selector(handleUnsupportedAppVersionError:bridgeNetworkManager:)] ||
+            ![appDelegate handleUnsupportedAppVersionError:error bridgeNetworkManager:self])
+        {
+            // Show default alert with a button tap to take the user to the app store to update
+            NSString *localizedTitle = NSLocalizedStringWithDefaultValue(@"SBB_ALERT_TITLE_UNSUPPORTED_APP", @"BridgeSDK", [NSBundle bundleForClass:[BridgeSDK class]], @"Unsupported App Version", @"Alert title: Unsupported App Version");
+            NSString *localizedDismiss = NSLocalizedStringWithDefaultValue(@"SBB_ALERT_DISMISS_BUTTON", @"BridgeSDK", [NSBundle bundleForClass:[BridgeSDK class]], @"Dismiss", @"Alert button: dismiss");
```

Comment:
I'd include more context for the translators, both in the key names and the comments, specifying that these buttons are in the unsupported app version alert.